### ISSUE TITLE
Right-click menu not appearing on Windows/Linux

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#2747]
Repro is with GA Zui [v1.0.1](https://github.com/brimdata/zui/releases/tag/v1.0.1).

A community user reported the following in a [Slack message](https://brimdata.slack.com/archives/C010DR0HHMF/p1681070991086319) and provided the attached video. I've reproduced the same behavior on Windows and also confirmed that it does _not_ seem to repro on macOS. 🤔  In their own words:

> Right clicking on the domain name may result in the context menu not appearing. A work around is to select the ‘inspector’ or clicking the corner of the table.
> This was was on a Windows machine : )

https://user-images.githubusercontent.com/5934157/230795018-d66d8bf8-2404-4b80-82f5-80097434b09c.mp4

**Comment by philrz** [comment]
Another community user (running Zui Insiders on Windows 10) reported on a [Slack thread](https://brimdata.slack.com/archives/C03MW6XT7HC/p1696528798214759) that they bumped into what looks like another instance this issue. In their own words:

> is this expected? my right click only works in table view if I right click on white space. If there is a value underneath the menu just blinks
> This is win 10
> ![zui_insiders_right_click](https://github.com/brimdata/zui/assets/5934157/cf5f1f1d-5d03-40bc-afaa-a0eaf90e9b7a)

I first tried to reproduce the problem with Zui Insiders [1.3.1-4](https://github.com/brimdata/zui-insiders/releases/tag/v1.3.1-4) on macOS using attached data [input.csv](https://github.com/brimdata/zui/files/12822192/input.csv) and found it would _not_ repro. I then tried on Windows 10 and found it _did_ repro. Finally, I tried on Linux and found it _did_ repro there, as shown in the attached video. Notice how sometimes after I right-click we can briefly see the ghostly outline of the menu.

https://github.com/brimdata/zui/assets/5934157/3c02c246-fb9c-4679-a307-6a6178610e26

The "it repros on Windows but not on macOS" symptom rang a bell and I found this existing issue. Since this is the second community user to independently bump into this, I'm boosting the priority in hopes it can be addressed before our next GA release.

**Comment by philrz** [comment]
Verified in Zui commit 9f091b6.

@jameskerr recently picked up this issue with intent to fix it, only to find that he could no longer reproduce it on Windows. Since it also repro'ed on Linux in the past, I looked into it further there and via binary search have confirmed that it started working at commit 9f091b6, which is associated with the changes in #2895. The attached video confirms that right-clicks on  the values now bring up the menu as well, not just when clicking on white space. The "rings" effect appears with each right-click.

https://github.com/brimdata/zui/assets/5934157/fcbfff76-a938-40a7-acbf-5bc4cdc6ef9a

Thanks @jameskerr!
